### DISCUSSION TITLE
Shadcn with Laravel Jetstream

### DISCUSSION THREAD

**Discussion by madebyitalo** [shadcn-ui/ui#3951] [Q&A]
I'm testing shadcn, I use Laravel a lot to develop applications. I checked that shadcn has an installation guide for laravel, but using Breeze starter kit. 

What is the best way to use shadcn with laravel, but with the Jetstream starter kit?

**Comment by renanwilliam** [comment]
https://github.com/renanwilliam/laravel-jetstream-with-spin-and-shadcn
